src/main/kotlin/dev/themeinerlp/attollo/listener/UpdateCheckerListener.kt

```diff
@@ -0,0 +1,19 @@
+package dev.themeinerlp.attollo.listener
+
+import dev.themeinerlp.attollo.Attollo
+import dev.themeinerlp.attollo.NOTIFY_UPDATE_PERMISSION
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.event.player.PlayerJoinEvent
+
+class UpdateCheckerListener(private val attollo: Attollo) : Listener {
+
+    @EventHandler
+    fun onJoin(event: PlayerJoinEvent) {
+        val player = event.player
+        if (player.isOp || player.hasPermission(NOTIFY_UPDATE_PERMISSION)) {
```

Comment:
Add an additional check to allow people to disable based on op the Notification

Reply:
What do you mean by this? Like add a config option where when disabled, it only sends it to people with a permission and not ops as well?

Reply:
I would like to force the admins as a specialist department to always receive a notification, no matter when. The only problem is that there are users who can have OP + permission.
Because OP comes from Vanilla and Permissions from Bukkit where the backend is e.g. Luckperms. 

So you should consider the case OP + Permission, OP only or Permission only